system/hardware/pc/hardware.h

```diff
@@ -15,7 +15,7 @@ class HardwarePC : public HardwareNone {
     volume = util::map_val(volume, 0.f, 1.f, MIN_VOLUME, MAX_VOLUME);
 
     char volume_str[6];
-    snprintf(volume_str, sizeof(volume_str), "%.3f", volume);
-    std::system(("pactl set-sink-volume @DEFAULT_SINK@ " + std::string(volume_str)).c_str());
+    snprintf(volume_str, sizeof(volume_str), "%.0f", volume * 100);
+    std::system(("pactl set-sink-volume @DEFAULT_SINK@ " + std::string(volume_str) + "%").c_str());
```

Comment:
the volume will be scaled "naturally" if the argument is a percentage - let pulseaudio do this here and remove the scaling from soundd?